src/Symfony/Component/AssetMapper/Command/ImportMapInstallCommand.php

```diff
@@ -42,18 +42,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $finishedCount = 0;
         $progressBar = new ProgressBar($output);
         $progressBar->setFormat('<info>%current%/%max%</info> %bar% %url%');
-        $downloadedPackages = $this->packageDownloader->downloadPackages(function (string $package, string $event, ResponseInterface $response, int $totalPackages) use (&$finishedCount, $progressBar) {
-            $progressBar->setMessage($response->getInfo('url'), 'url');
-            if (0 === $progressBar->getMaxSteps()) {
-                $progressBar->setMaxSteps($totalPackages);
-                $progressBar->start();
-            }
 
-            if ('finished' === $event) {
-                ++$finishedCount;
-                $progressBar->advance();
-            }
-        });
+        try {
+            $downloadedPackages = $this->packageDownloader->downloadPackages(function (string $package, string $event, ResponseInterface $response, int $totalPackages) use (&$finishedCount, $progressBar) {
+                $progressBar->setMessage($response->getInfo('url'), 'url');
+                if (0 === $progressBar->getMaxSteps()) {
+                    $progressBar->setMaxSteps($totalPackages);
+                    $progressBar->start();
+                }
+
+                if ('finished' === $event) {
+                    ++$finishedCount;
+                    $progressBar->advance();
+                }
+            });
+        } catch (\Throwable $throwable) {
```

Comment:
Do we really need to catch all `Throwable` instances? Isn't there any specific exception we can catch?

Reply:
I will change it to the LogicException. Thank you for the feedback.